ticket_dispenser.c

```diff
@@ -1,149 +1,203 @@
-#include <Wire.h>
-
+#include "Wire.h"
 #include "Keypad.h"
 #include "Adafruit_LEDBackpack.h"
 #include "Adafruit_GFX.h"
 
 Adafruit_7segment matrix = Adafruit_7segment();
-
+//
 // Setup the keypad
-const byte ROWS = 4; //four rows
-const byte COLS = 3; //three columns
+//
+const byte ROWS = 4;
+const byte COLS = 3;
 char keys[ROWS][COLS] = {
```

Comment:
That would be an interesting move... but I'm not sure where to begin there. If you have any specific ideas or examples please share!